hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2256,6 +2256,17 @@ void cloneTableSchema(TableName tableName, TableName newTableName, boolean prese
    */
   boolean isCompactionOffloadEnabled() throws IOException;
 
+  /**
+   * update compaction server total throughput bound
+   * @param upperBound the total throughput upper bound of all compaction servers
```

Comment:
When we talk of compaction servers here, are we talking about the new compaction server facility or are we talking about regionservers?

Reply:
we can control compaction throughtput on compactionServers, can not control regionservers

Reply:
Most deploys will not use a compaction server suite. Is it correct then to talk of compaction servers in our Admin API? Should there be another channel for interacting with compaction servers? A CompactionServersAdmin ?

Reply:
This method talk to master only, compaction servers get throughput control message through periodic heartbeat report